lib/methods/get-community.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+// node core modules
+
+// 3rd party modules
+const _ = require('lodash');
+
+// internal modules
+const { omitDefaultRequestParams, constructError } = require('./utils');
+const { getCommunity: parser } = require('../response-parsers');
+
+const qsValidParameters = ['communityUuid'];
+
+/**
+ * Get all details for a community given its id.
+ *
+ * @param  {String}   communityUuid     URL parameter that is unique for every community.
+ * @param  {Object}   options           Additional information required by building http request.
+ * @param  {Function} callback          Called with `(err, community)`
```

Comment:
can you pls specify what the callback arg `community` looks like?
take a look at jsdoc global typedef (e.g. in [oniyi-http-client](https://github.com/benkroeger/oniyi-http-client/blob/master/lib/make-request.js#L13))

Reply:
I believe, you can define [callback](http://usejsdoc.org/tags-callback.html) as global typedef and also define it's invocation args